categories/categories.go

```diff
@@ -0,0 +1,48 @@
+package categories
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/golang/glog"
+	"github.com/prebid/prebid-server/stored_requests"
+)
+
+type Categories struct {
```

Comment:
You could make Categories a simple type:
`type Categories map[string]map[string]map[string]string`

Then below instead of referencing `c.Categories[primaryAdServer]` it would just be `c[primaryAdServer]`

If you have potential plans to store more than just this map in the Categories struct, then leaving it as a struct gives you that future flexibility.

Reply:
@hhhjort we may have some other (meta)data in categories. Also there is a function 
func (c *Categories) GetCategory() that uses data from Categories struct.
I think  struct simplifies code. If we will need to change categories data format, we will not need to change it in a lot of places in function signatures.
Please let me know how do you think it should be.